index.html

```diff
@@ -1,298 +1,348 @@
+
+            <div class="lang-box">
+                <div class="lang-logo-html"></div>
+                <div class="lang-label">HTML</div>
             </div>
-        
 
-        <div class="social-area">
-            <h4 class="social-network">LET’S CONNECT</h4>
-            <div class="social-icon">
+            <div class="lang-box">
+                <div class="lang-logo-css"></div>
+                <div class="lang-label">CSS</div>
+            </div>
+
+        </div>
+        <div class="framework-select">
+            <div class="framework-label">Framework</div>
+            <div class="frameW-arrow-button"><a href="#"><img alt=" "
+                        src="../setup-and-mobile-version-skeleton/images/icons8-right-arrow-32.png"></a></div>
+        </div>
+
+        <div class="skills-select">
+            <div class="skills-label">Skills</div>
+            <div class="skills-arrow-button"><a href="#"><img alt=" "
+                        src="../setup-and-mobile-version-skeleton/images/icons8-right-arrow-32.png"></a></div>
+        </div>
+
+    </section>
+    <section>
+        <div class="form-container">
+            <div class="top-half-circle contact-form-imgs"></div>
+            <div class="top-rectangle contact-form-imgs"></div>
+            <div class="top-circle contact-form-imgs"></div>
+            <div class="bottom-rectangle contact-form-imgs"></div>
+            <div class="bottom-circle contact-form-imgs"></div>
+            <h2>Contact me</h2>
+            <p>If you have an application you are interested in developing,
+                a feature that you need built or a project that needs coding.
+                I’d love to help with it </p>
+
+            <form action="https://formspree.io/f/xdobpeyw" method="post">
                 <ul>
                     <li>
-                        <a href="#" class="twitter"><img src="../setup-and-mobile-version-skeleton/images/Twiter.svg" alt=""></a>
-                    </li>
-                    <li>
-                        <a href="#" class="linkedin"><img src="../setup-and-mobile-version-skeleton/images/Linkedin.svg" alt=""></a>
+                        <input id="name" type="text" placeholder="Please enter your name" name="user_name" value=""
+                            maxlength="30" required>
                     </li>
+
                     <li>
-                        <a href="#" class="M"><img src="../setup-and-mobile-version-skeleton/images/M.svg" alt=""></a>
+                        <input type="email" name="user_email" required placeholder="Please enter your email">
+
                     </li>
                     <li>
-                        <a href="#" class="C"><img src="../setup-and-mobile-version-skeleton/images/C.svg" alt=""></a>
+                        <textarea id="message" name="ueser_message" placeholder="Write your message here"
+                            maxlength="500"></textarea>
```

Comment:
Kindly check the spelling of the `name` attribute. You want other users to understand your code.
Also, your `textarea` ☝🏽  input has to be `required`. Kindly add `required` so that the form cannot be submitted without the field having content.